.github/ISSUE_TEMPLATE/icon_request.yml

```diff
@@ -55,6 +55,20 @@ body:
     validations:
       required: true
 
+  - type: checkboxes
+    attributes:
+      label: Forbidden Brands
+      options:
+        - label: I have reviewed the list of [forbidden brands](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#forbidden-brands) and can confirm the brand I am requesting is not one of them.
```

Comment:
Should we _explicitly_ mention subsidiaries here? 🤔

Reply:
Sure, feel free to suggest a wording change :)